python/seldon_deploy_sdk/api_client.py

```diff
@@ -61,20 +61,22 @@ class ApiClient(object):
     }
 
     def __init__(self, configuration=None, header_name=None, header_value=None,
-                 cookie=None):
+                 cookie=None, authenticator=None):
         if configuration is None:
             configuration = Configuration()
         self.configuration = configuration
 
         # Use the pool property to lazily initialize the ThreadPool.
         self._pool = None
+        self.authenticator = authenticator
         self.rest_client = rest.RESTClientObject(configuration)
         self.default_headers = {}
         if header_name is not None:
             self.default_headers[header_name] = header_value
         self.cookie = cookie
         # Set default User-Agent.
         self.user_agent = 'Swagger-Codegen/0.2.1/python'
+        self.client_side_validation = configuration.client_side_validation
```

Comment:
What's the `client_side_validation` flag used for? Is that something coming from a newer version of `swagger-codegen`?

Reply:
Yeah just copied it from the swagger templates